lib/modules/manager/copier/artifacts.spec.ts

```diff
@@ -122,6 +122,23 @@ describe('modules/manager/copier/artifacts', () => {
       ]);
     });
 
+    it('invokes copier update with nested destination and answer file', async () => {
```

Comment:
Please change the PR title to:

`fix(manager/copier): process update templates outside repository root`